docker/docker-compose.test.yml

```diff
@@ -74,22 +123,531 @@ services:
+      STACKS_CORE_EVENT_PORT: 3700
+      STACKS_CORE_EVENT_HOST: "0.0.0.0"
+      STACKS_BLOCKCHAIN_API_PORT: 3999
+      STACKS_BLOCKCHAIN_API_HOST: "0.0.0.0"
+      STACKS_CORE_RPC_HOST: "stacks-node"
+      STACKS_CORE_RPC_PORT: 20443
+      API_DOCS_URL: http://localhost:3999/doc
+    profiles:
+      - default
+
+  stacks-explorer:
+    image: stacks-explorer
+    container_name: stacks-explorer
+    stop_grace_period: 5s
+    build:
+      context: stacks-explorer
+      args:
+        GIT_URI: "https://github.com/hirosystems/explorer.git"
+        GIT_BRANCH: "v1.245.0"
+    ports:
+      - 3020:3000
+    depends_on:
+      - bitcoin
+      - stacks-node
+      - stacks-api
+      - postgres-stacks-api
+    environment:
+      NEXT_PUBLIC_MAINNET_API_SERVER: "http://stacks-api:3999"
+    profiles:
+      - default
+
+  # Mempool.
+  # --------
+  electrs:
+    image: electrs:latest
+    container_name: electrs
+    stop_grace_period: 5s
+    build: electrs
+    ports:
+      - 60401:60401
+      - 3002:3002
+    depends_on:
+      - bitcoin
+      - bitcoin-miner
+    environment:
+      RUST_BACKTRACE: 1
+      BITCOIN_RPC_HOST: "bitcoin"
+      BITCOIN_RPC_PORT: "18443"
+    profiles:
+      - bitcoin-mempool
+
+  mariadb:
+    image: mariadb:10.5.21
+    container_name: mariadb
+    stop_grace_period: 5s
+    ports:
+      - 3306:3306
+    environment:
+      MYSQL_DATABASE: "mempool"
+      MYSQL_USER: "mempool"
+      MYSQL_PASSWORD: "mempool"
+      MYSQL_ROOT_PASSWORD: "admin"
+    profiles:
+      - bitcoin-mempool
+
+  mempool-web:
+    image: mempool/frontend:latest
+    container_name: mempool-web
+    stop_grace_period: 5s
+    depends_on:
+      - mempool-api
+      - mariadb
+      - electrs
+    user: "1000:1000"
+    restart: on-failure
+    ports:
+      - 8083:8083
+    environment:
+      FRONTEND_HTTP_PORT: "8083"
+      BACKEND_MAINNET_HTTP_HOST: "mempool-api"
+    command: "./wait-for mariadb:3306 --timeout=720 -- nginx -g 'daemon off;'"
+    profiles:
+      - bitcoin-mempool
+
+  mempool-api:
+    image: mempool/backend:latest
+    container_name: mempool-api
+    stop_grace_period: 5s
+    depends_on:
+      - electrs
+      - mariadb
+    user: "1000:1000"
+    restart: on-failure
+    ports:
+      - 8999:8999
+    environment:
+      # Connect to electrs host
+      MEMPOOL_BACKEND: "electrum"
+      ELECTRUM_HOST: "electrs"
+      ELECTRUM_PORT: "60401"
+      ELECTRUM_TLS_ENABLED: "false"
+      # Connect to bitcoin rpc
+      CORE_RPC_HOST: "bitcoin"
+      CORE_RPC_PORT: "18443"
+      CORE_RPC_USERNAME: "devnet"
+      CORE_RPC_PASSWORD: "devnet"
+      DATABASE_ENABLED: "true"
+      DATABASE_HOST: "mariadb"
+      DATABASE_DATABASE: "mempool"
+      DATABASE_USERNAME: "mempool"
+      DATABASE_PASSWORD: "mempool"
+      STATISTICS_ENABLED: "true"
+    command: "./wait-for-it.sh mariadb:3306 --timeout=720 --strict -- ./start.sh"
+    profiles:
+      - bitcoin-mempool
+
+  bitcoind:
```

Comment:
There's already a bitcoin container at the beginning, why do we need a separate one?